app/views/content/train-to-be-a-teacher/initial-teacher-training.md

```diff
@@ -53,15 +53,17 @@ environment
 
 ## Your classroom placements
 
-Typically, you will get 120 days of classroom experience in a minimum of two schools. Through your placements, you’ll have the opportunity to teach, plan, observe and research.
+You should get 120 days of classroom experience as part of a postgraduate or 3 year undergraduate teacher training course.
```

Comment:
```suggestion
You should get 120 days of classroom experience as part of a postgraduate teacher training course.
```
I know you've removed the 4 year undergrad reference but shouldn't we remove the 3 year one too @philspray ?

I'll raise a separate ticket for post cycle start about making it clearer when we are talking about postgrad around the site in general, as suggested by @EllieNodder